src/library/mima-filters/2.13.0.forwards.excludes

```diff
@@ -0,0 +1,2 @@
+ProblemFilters.exclude[DirectMissingMethodProblem]("scala.sys.process.ProcessIO.connectInput")
+ProblemFilters.exclude[DirectMissingMethodProblem]("scala.sys.process.ProcessIO.this")
```

Comment:
please keep backward and **forward** compatibility

Reply:
Let me be clear, `ProcessIO` is `final` here. My intuition is that we can safely add class members because we don't have to worry about name collision in the classes inheriting `ProcessIO`, isn't it right?

If that's not forward-compatible, I will have no choice other than to wait until 2.14.x branch is opened...

Reply:
I think no difference whether the class is final or not. Adding method is just **NOT** forward binary compatible.

Reply:
that is correct. if you add a method, compile against the new version, and then run against the old version, it will have a linkage error.